tests/functional/test_cli.py

```diff
@@ -148,3 +148,14 @@ def test_network_option_adhoc(runner, network_cmd, network_input):
     result = runner.invoke(network_cmd, ["--network", network_input])
     assert result.exit_code == 0, result.output
     assert OUTPUT_FORMAT.format(network_input) in result.output
+
+
+def test_network_option_make_required(runner):
+    @click.command()
+    @network_option(required=True)
+    def cmd(network):
```

Comment:
@johnson2427 we can make the `ape cache init` and `ape cache purge` etc. commands set `required=True` so that the user _has_ to provide a network. This will lessen the confusionz because

1. The user will only get the local network error message if they explicitly try to use the local network and not when excluding the option altogether
2. The user when excluding the option will realize they need the option and then likely input them mainnet value or something and never ever hit the local-error network to begin with

Reply:
Was just thinking about this today